scripts/release-api.sh

```diff
@@ -22,7 +22,6 @@ fi
 
 echo "Setting API version to ${RELEASE_API_VERSION}"
 mvn -q versions:set -DnewVersion="${RELEASE_API_VERSION}" -DprocessAllModules=true -pl ":kroxylicious-api" -DgenerateBackupPoms=false
```

Comment:
```suggestion
mvn -q versions:set -DnewVersion="${RELEASE_API_VERSION}" -DprocessAllModules=true -pl ${API_MODULES} -DgenerateBackupPoms=false
```

Though arguably we should just do away with the variable.